doc/qbk/07_concepts/BuffersGenerator.qbk

```diff
@@ -0,0 +1,128 @@
+[/
+    Copyright (c) 2016-2022 Vinnie Falco (vinnie dot falco at gmail dot com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/boostorg/beast
+]
+
+[section:BuffersGenerator BuffersGenerator]
+
+A [*BuffersGenerator] provides a generalized interface for generating
+serialized data for sequential processing.
+
+The generator will be asked to produce buffers. The consuming code will signal
+how much of the data has been consumed, and repeatedly query for buffers until
+no more data is available, or the generator indicates an error condition.
+
+In this way, serializers can be adapted as [*BuffersGenerator], for example
+__message_generator__ which provides a type-erased interface for a variety of
+concrete http message types.
+
+Overloads of `write` and `async_write` operations are provided as free
+functions. These operations will consume the output of a [*BuffersGenerator]
+and process the data by writing them to a __SyncWriteStream__ or
+__AsyncWriteStream__ respectively.
+
+[heading Associated Types]
+
+* [link beast.ref.boost__beast__core__is_buffers_generator `is_buffers_generator`]
+* __ConstBufferSequence__
+
+[heading Requirements]
+
+[warning
+    These requirements may undergo non-backward compatible
+    changes in subsequent versions.
+]
+
+In this table:
+
+* `G` denotes a type meeting the requirements of [*BuffersGenerator].
+* `g` denotes a value of type `G`.
+* `n` is a value of type `std::size_t`.
+* `ec` is a value of type [link beast.ref.boost__beast__error_code `error_code&`].
+
+[table Valid expressions
+[[Expression] [Type] [Semantics, Pre/Post-conditions]]
+[
+    [`G::const_buffers_type`]
+    [`implementation-defined`]
```

Comment:
You don't need to say implementation-defined here, you should just leave the column empty because we are already naming a type.

And if you were going to say _implementation-defined_ it should be in italics not a code font.

Reply:
Got it